_data/projects/gkv.yml

```diff
@@ -0,0 +1,12 @@
+name: Gkv
+desc: An open source, git-based key value store for small, simple projects
+site: https://github.com/ybur-yug/gkv
+tags:
+  - oss
+  - databases
+  - ruby
+  - db
+  - kv
+upforgrabs:
+  name: Gkv
+  link: https://github.com/ybur-yug/gkv
```

Comment:
This link should be to the list of issues which are recommended for new contributors, e.g:

https://github.com/ybur-yug/gkv/labels/help%20wanted

`help wanted` then becomes the `name` value above...